### PR TITLE
chore(manager): move the manager and rename the repository to `outline_apps`

### PR DESCRIPTION
**Pull request by daniellacosse**
This way we'll be able to share web code and build scripts, and tools between the projects (like the storybook!)

I also have a draft PR up that moves the client into its own folder! https://github.com/Jigsaw-Code/outline-client/pull/1838

### CODE REVIEW

File: server_manager/electron_app/fetch.ts [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+
+import type {HttpRequest, HttpResponse} from '../infrastructure/path_api';
+
+export const fetchWithPin = async (req: HttpRequest, fingerprint: string): Promise<HttpResponse> => {
+  const response = await new Promise<IncomingMessage>((resolve, reject) => {
+    const options: https.RequestOptions = {
+      ...urlToHttpOptions(new URL(req.url)),
+      method: req.method,
+      headers: req.headers,
+      rejectUnauthorized: false, // Disable certificate chain validation.
```
[the remark this answers is not in the document]
Comment (fortuna) [reply server_manager/electron_app/fetch.ts:29]: We validate with the certificate fingerprint

File: .gitignore [hunk 2]
```diff
@@ -26,3 +27,5 @@ tools/smartdnsblock/bin/*
 !tools/smartdnsblock/bin/*.exe
 universal.apk
 xcuserdata/
+do_install_script.ts
```
Comment (fortuna) [inline .gitignore:30]: Is this in the root?

I'd rather keep the .gitignore localized in the respective folder.

File: package.json [hunk 3]
```diff
@@ -167,6 +167,7 @@
     ]
   },
   "workspaces": [
-    "src/cordova/plugin"
+    "src/cordova/plugin",
```
Comment (fortuna) [inline package.json:170]: I thought you said there would be one workspace?

**Review by fortuna — COMMENTED** [review]
Of the files you are adding, which ones did you modify from the original? It's impossible to tell without a diff.

Also, please make sure you included all the recent changes:
https://github.com/Jigsaw-Code/outline-server/pulls?q=is%3Apr+is%3Aclosed

**Comment by sbruens** [comment]
Could we block further merges on the manager?

File: server_manager/install_scripts/install_server.sh
[the remark this answers is not in the document]
Comment (fortuna) [reply server_manager/install_scripts/install_server.sh:1]: See https://github.com/search?q=repo%3AJigsaw-Code%2Foutline-server%20install_script&type=code

Most critically, we have it on the UI: https://github.com/Jigsaw-Code/outline-server/blob/a62b25b54cf97770645d3fb601f601d2c92006be/src/server_manager/web_app/ui_components/outline-manual-server-entry.ts#L328

So we will have to keep it there for some time until everyone migrate. But also update all the docs and references.

File: .gitignore (hunk 2 above)
Comment (fortuna) [reply .gitignore]: Please create a .gitignore in the directory where do_install_script.ts is created, so it's localized there.
I'd say same for `/server_manager` to ignore the node_modules.

It's easier to manage these if they are in context.

**Review by fortuna — COMMENTED** [review]
I really need to know which, of the added files, you edited, so I know what to review.

**Comment by fortuna** [comment]
By the way, were you able to successfully run the manager and create a server?

**Comment by sbruens** [comment]
> > Could we block further merges on the manager?
> 
> Like, just not merge anything that touches the manager? Or write a check to enforce that?

I don't think we need to formalize it with a check, we're small enough to manage that informally? Unless it's easy to configure in GitHub somehow.

**Comment by fortuna** [comment]
@daniellacosse I just need to know the places where you changed code, ignore formatting.
For sure you had to edit some files to make the code work. We need to know what you had to modify.

If you had branches after the copy, but before and after the changes, it would be ideal. Otherwise, knowing the files you changed would help.

BTW, the PR title needs to be updated.

File: server_manager/tsconfig.json [hunk 4]
```diff
@@ -0,0 +1,24 @@
+{
```
Comment (fortuna) [inline server_manager/tsconfig.json:1]: This seems to match the original behavior. We should consider factoring out the common settings across projects at some point, like we have in outline-server.

**Review by fortuna — APPROVED** [review]
Looks good. Thanks for the diff, I took a look at the most relevant files that changed and all looks good.

Please make sure the app still works with no runtime errors.

**Pull request merged**